cogito/buildinfo.go

```diff
@@ -1,11 +1,11 @@
 package cogito
 
 // Baked in at build time with the linker. See the Taskfile and the Dockerfile.
-var buildinfo = "unknown"
+var buildInfo = "unknown"
```

Comment:
this required a change in:
https://github.com/Pix4D/cogito/blob/9558fa3fe5454cc1e1ff44c144cef9786862485b/Taskfile.yml#L22
https://github.com/Pix4D/cogito/blob/9558fa3fe5454cc1e1ff44c144cef9786862485b/Dockerfile#L24

Reply:
This was reverted in 558f12d778497501ba0122ea58436672971dd71c. It was gong to be a larger change than I expected, yet it has nothing to do with GHE.